packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -16,11 +16,31 @@ interface AutocompleteProps {
    */
   readonly value: Option | undefined;
 
+  /**
+   * Hint text that goes above the value once the form is filled out.
+   */
+  readonly menuSize?: "medium";
+
   /**
    * Hint text that goes above the value once the form is filled out.
    */
   readonly placeholder: string;
 
+  /**
+   * Turn on separators between result elements.
+   */
+  readonly separators?: boolean;
+
+  /**
+   * Turn off checkmarks on selected result elements.
+   */
+  readonly checkmarks?: boolean;
```

Comment:
What is the reason to remove these?

Reply:
for autocomplete being used as a search it doesn't fit our designs. 
ex. i search for a specific product the only result in the list is the item that fits the search, and when the drop down opens again the only results would be the search results 
The case where you select something from the list and then open the entire list again doesn't apply to the use case so the checkmarks don't make sense
[image](https://user-images.githubusercontent.com/49168885/88966964-6456d500-d26a-11ea-85c5-0196bbb665ac.png)

Reply:
If you clear the value (I assume you're doing this) after selecting the item from the menu the checkmark should also be removed?

Reply:
We'll loop in with Josh / Rebeca on Tuesday.